cmd/operate/env.go

```diff
@@ -0,0 +1 @@
+package operate
```

Comment:
why do we need this?

Reply:
this is for future development

Reply:
we dont have a env operation yet, even in the existing product. Lets remove it, we will add when we need it.

Reply:
@dhavalmehta07 we have env operations